src/App.css

```diff
@@ -10,3 +10,15 @@
   border: 3px solid black;
   width: 700px;
 }
+
```

Comment:
- [ ] It would be better if we update the README to include the following; technologies used, how to clone and run the app locally, author section, etc. I think this is a great time to replace the `Added soon` text with the right information. Our README file must be professional in order for approval